enterprise/api/v1/views.py

```diff
@@ -565,36 +568,54 @@ def post(self, request):
             The name of the enterprise requesting more codes.
         *number_of_codes*
             The number of codes requested.
+        *notes*
+            Help notes related to codes request.
         """
         try:
-            email, enterprise_name, number_of_codes = self.get_required_query_params(request)
+            email, enterprise_name, number_of_codes, notes = self.get_required_query_params(request)
         except CodesAPIRequestError as invalid_request:
             return Response({'error': str(invalid_request)}, status=HTTP_400_BAD_REQUEST)
 
         subject_line = _('Code Management - Request for Codes by {token_enterprise_name}').format(
             token_enterprise_name=enterprise_name
         )
-        msg_with_codes = _('{token_email} from {token_enterprise_name} has requested {token_number_codes} additional '
-                           'codes. Please reach out to them.').format(
-                               token_email=email,
-                               token_enterprise_name=enterprise_name,
-                               token_number_codes=number_of_codes)
-        msg_without_codes = _('{token_email} from {token_enterprise_name} has requested additional codes.'
-                              ' Please reach out to them.').format(
-                                  token_email=email,
-                                  token_enterprise_name=enterprise_name)
+        if number_of_codes and notes:
```

Comment:
Can we extract out this code to a create_message_body function?

Reply:
I have added a new function named create_message_body. please take a look.